pep-0482.txt

```diff
@@ -195,67 +214,7 @@ Example code with types::
   }
 
 
-References
-==========
-
-.. [mypy]
-   http://mypy-lang.org
-
-.. [reticulated]
-   https://github.com/mvitousek/reticulated
-
-.. [reticulated-paper]
-   http://wphomes.soic.indiana.edu/jsiek/files/2014/03/retic-python.pdf
-
-.. [pycharm]
-   https://github.com/JetBrains/python-skeletons#types
-
-.. [obiwan]
-   http://pypi.python.org/pypi/obiwan
-
-.. [numba]
-   http://numba.pydata.org
-
-.. [pytypedecl]
-   https://github.com/google/pytypedecl
-
-.. [argumentclinic]
-   https://docs.python.org/3/howto/clinic.html
-
-.. [numpy]
-   http://www.numpy.org
-
-.. [typescript]
-   http://www.typescriptlang.org
-
-.. [hack]
-   http://hacklang.org
-
-.. [dart]
-   https://www.dartlang.org
-
-.. [actionscript]
-   http://livedocs.adobe.com/specs/actionscript/3/
-
-.. [pyflakes]
-   https://github.com/pyflakes/pyflakes/
-
-.. [pylint]
-   http://www.pylint.org
-
-
 Copyright
 =========
 
 This document has been placed in the public domain.
-
-
-
-..
-   Local Variables:
-   mode: indented-text
-   indent-tabs-mode: nil
-   sentence-end-double-space: t
-   fill-column: 70
-   coding: utf-8
-   End:
```

Comment:
Whilst we're editing these files, please could you also remove this redundant emacs metadata from the others too?


```console
❯ git grep "Local Variables:" pep-0394.txt pep-0400.txt pep-0482.txt pep-0526.txt pep-0544.txt pep-0557.rst pep-0573.rst pep-0583.rst pep-0612.rst pep-0615.rst pep-0622.rst pep-0633.rst pep-0634.rst pep-0636.rst pep-0642.rst pep-0649.rst pep-0669.rst pep-0675.rst pep-0680.rst pep-0689.rst pep-0692.rst pep-0695.rst pep-0696.rst pep-0697.rst pep-0701.rst pep-0704.rst pep-0710.rst pep-0720.rst pep-3151.txt
pep-0544.txt:   Local Variables:
pep-0557.rst:   Local Variables:
pep-0573.rst:   Local Variables:
pep-0583.rst:   Local Variables:
pep-0615.rst:   Local Variables:
pep-0622.rst:   Local Variables:
pep-0636.rst:   Local Variables:
pep-0669.rst:    Local Variables:
pep-0675.rst:   Local Variables:
pep-0680.rst:    Local Variables:
pep-3151.txt:   Local Variables:
```